uber/templates/base.html

```diff
@@ -205,7 +150,39 @@
                             </ul>
                         </li>
                     </ul>
-                    <ul class="nav navbar-nav" id="main-menu"></ul>
+                    {% block mainmenu %}
+                    <ul class="nav navbar-nav" id="main-menu">
+                        {% block mainmenu_items %}
+                        {% for menu_item in c.MENU_JSON_DECODED %}
+                            {% if menu_item.href %}
+                                <li><a href="{{ menu_item.href }}">{{ menu_item.name }}</a></li>
+                            {% elif menu_item.submenu %}
```

Comment:
There is no `else` case for handling what will happen if a top-level menu item doesn't have a submenu OR a link.